source/documentation/fuseki2/fuseki-config-endpoint.md

```diff
@@ -146,13 +173,15 @@ A GSP operation has `?default` or `?graph=`.
 Quads operations have no query string and a have a `Content-Type` for a data in
 a RDF triples or quads syntax.
 
+Quads operations only apply to requests sen tot he dataset URL.
```

Comment:
s/sen tot he/sent to the/

Reply:
(The statement isn't actually correct! I just tested it and quads work on a named endpoint.)